packages/x-components/src/components/decorators/__tests__/bus.decorators.spec.ts

```diff
@@ -42,22 +42,22 @@ describe('testing @XOn decorator', () => {
       dataListener(this, payload);
```

Comment:
I'm not sure if the rename of `UserClickedOpenX`, and  `UserClickedCloseX` with those generic events is correct.
I think they're different events but tell me if I'm wrong.

Reply:
In this test makes sense to have the specific events.

Reply:
The thing here is that I was taking a look with Ivan and we show that these events are related to opening a modal, not X, I mean, if we use the modals for other purposes, we are launching the UserClickedOpenX event when you open an Events Modal, and it doesn't make sense. 

So we think that we have to rename the events to have a standard name that refers to close the modal itself instead of closing X components.